coriolis/osmorphing/osmount/base.py

```diff
@@ -233,9 +243,8 @@ def _get_volume_block_devices(self):
             "lsblk -lnao KNAME").decode().split('\n')[:-1]
         LOG.debug("All block devices: %s", str(volume_devs))
 
-        # Skip this instance's current root device (first in the list)
         volume_devs = ["/dev/%s" % d for d in volume_devs if
-                       not re.match(r"^.*\d+$", d)][1:]
```

Comment:
Great catch, I wasn't even aware we do this here (I presumed there was no filtering which is why I added the `ignore_devices` logic above)